Land_Sector_Data_Analysis/Land_Sector_Dataset_Description.md

```diff
@@ -0,0 +1,306 @@
+
+### 4.2.3 Planted forests:
+
+This data is from The Spatial Database of Planted Trees(SDPT). It contains data on 43 countries. Each file has the following properties:
+**final_id, iso, country, org_name, common_name, species, species_simp, plant_ag, timber_ag, ever_dec, conifer_broad, hard_soft, size, source, year, geometry**
+
+### 4.2.4 Wood Fiber Concessions:
+
+Contains files on the wood fiber concessions of 03 countries and 01 file for the world. The files have the following properties:
+**OBJECTID\*1. OBJECTID. group_comp. NAME, type, COUNTRY, area_ha, source, Last_updat, gfwid, Shape_Leng, source_typ, year, FID\*, source_t_1, Shape_Le_1,, Shape_Le_2, Shape_Length, Shape_Area, geometry.**
+
+## 4.3 Forest Loss
+
+Forest Loss contains 03 subdirectories:
+
+### 4.3.1 Hansen Global Forest Change:
+
+This directory contains 04 principal subdirectories(**datamask, gain, lossy year, and tree cover**). In these directories are GeoTiff tiles with the names implying the interest
+
+### 4.3.2 Intact Forest Landscapes:
+
+This directory contains zips and GeoJSON files for Intact forest landscapes. The notation of the files carries the year of recording.
+
+### 4.3.3 Terra i Forest Loss:
+
+It contains the terra i forest loss data for the country Uganda.
+
+## 4.4 Land Use
+
+This directory contains Global Food Security-Support Analysis Data. The subdirectories contain tiled files with GFSAD Global Crop Land Crop Extent data. These files notations contain the year as well as the regions of interest which are: 
+- Africa( GFSAD30AFCE_2015_resampledTiles), 
+- Australia-New_Zealand-China-Mongolia( GFSAD30AUNZCNMOCE_2015_resampledTiles), 
+- Europe-Centra_Asia-Russia-Middle_East( GFSAD30EUCEARUMECE_2015_resampledTiles), 
+- North America( GFSAD30NACE_2010_resampledTiles), 
+- South America( GFSAD30SACE_2015_resampledTiles) and 
+- Southeast-and-Northeast Asia( GFSAD30SEACE_2015_resampledTiles).
+
+# 5. Soil
+
+This directory contains 03 subdirectories and 02 files( Global soil organic carbon). The three subdirectories are:
+
+## 5.1 Harmonised World Soil Database(HWSD)
+
+This directory contains data tables with the following properties:
+
+ID MU_GLOBAL MU_SOURCE1 MU_SOURCE2 ISSOIL SHARE SEQ SU_SYM74 SU_CODE74 SU_SYM85 SU_CODE85 SU_SYM90 SU_CODE90 T_TEXTURE DRAINAGE REF_DEPTH AWC_CLASS PHASE1 PHASE2 ROOTS IL SWR ADD_PROP T_GRAVEL T_SAND T_SILT T_CLAY T_USDA_TEX_CLASS T_REF_BULK_DENSITY T_BULK_DENSITY T_OC T_PH_H2O T_CEC_CLAY T_CEC_SOIL T_BS T_TEB T_CACO3 T_CASO4 T_ESP T_ECE S_GRAVEL S_SAND S_SILT S_CLAY S_USDA_TEX_CLASS S_REF_BULK_DENSITY S_BULK_DENSITY S_OC S_PH_H2O S_CEC_CLAY S_CEC_SOIL S_BS S_TEB S_CACO3 S_CASO4 S_ESP S_ECE
+
+### 5.2 Soil Quality Crop Production
+
+Contains a zip file of the HWSD Soil Quality Crop Production.
+
+### 5.3 World Soil Resources(WSR)
+
+This directory contains 02 subdirectories:
+
+### 5.3.1 WSR by Country:
+
+Contains files whose notation indicates the country of interest. The properties are the same as in 1.1.1.
```

Comment:
It's probably easier to move the description of files with `_WSR` suffix to this section, rather than refer the reader back to the top.

Reply:
That's true. I would update the file.

Regards...